adapters/wsa/wsa_configuration.php

```diff
@@ -162,9 +162,12 @@ function get_running_conf()
     $login = $sms_sd_ctx->getLogin();
     $passwd = $sms_sd_ctx->getPassword();
 
+    $SD = &$network->SD;
```

Comment:
`$network` is not initialized.

Reply:
fix by this commit https://github.com/openmsa/Adapters/commit/38a1123dd0717365f493b27ae08b7561d111d3c8